### DISCUSSION TITLE
WPF xaml designer equivalent for Fluent UI Blazor used with MAUI/Blazor?

### DISCUSSION THREAD

**Discussion by learntocode66** [microsoft/fluentui-blazor#475] [Q&A] [answered]
Is there something like WPF xaml designer that can be used with Fluent UI Blazor in MAUI/Blazor hybrids? Thanks!

**Comment by vnbaaij** [comment, marked as answer]
No, nothing like that exists for the library or Blazor in general.

**Reply by learntocode66** [reply]
Thank you!
